mavlink/message.go

```diff
@@ -32,6 +32,8 @@ var (
 type Message interface {
 	Pack(*Packet) error
 	Unpack(*Packet) error
+	MsgID() uint8
```

Comment:
Mavlink v2 extended message set is deliberately ignored here for the moment. There's a plan to replace all references to uint8 where message id is meant to specific MessageID type, but not now.